erpnext_shopify/sync_orders.py

```diff
@@ -137,7 +137,7 @@ def get_order_items(order_items, shopify_settings):
 			"item_name": shopify_item.get("name"),
 			"rate": shopify_item.get("price"),
 			"qty": shopify_item.get("quantity"),
-			"stock_uom": shopify_item.get("sku"),
+			"stock_keeping_unit": shopify_item.get("sku"),
```

Comment:
Did you added  `stock_keeping_unit` as a custom field in Sales Order Item table ?